sandbox/src/services/mockResponseProvider.js

```diff
@@ -153,6 +153,12 @@ module.exports = {
   getExampleResponseForRetrieveBusinessFunctions: function () {
 
     return { responsePath: 'retrieveBusinessFunctions/responses/PractitionerRoleBundle.json', responseCode: 200 }
+  },
+
+
+  getExampleResponseForGeneratePatientLetter: function () {
+
+    return { responsePath: 'generatePatientLetter/responses/000000070000_Appointment_Confirmation_Summary_20210603121353.pdf', filename: '000000070000_Appointment_Confirmation_Summary_20210603121353.pdf', responseCode: 200 }
   }
 
```

Comment:
Should we have an example for each of the patient letters that we can generate : 
"Appointment Confirmation Summary"
"Appointment Request Summary" - Deferral
"Triage Summary"
"Appointment Request Summary" - Booking

we could return different pdfs depending on the ubrn

Update:
I've just tried using the sandbox in the VDI - which does not render the returned pdf.
Is there anyway we can set the content type in the sandbox response ?

Reply:
We could update it to return more pdf's. I'm just working towards the simplest solution, which is covering the examples requested by Ken and approved by business (which indicate single file is sufficient)

Yeah preview doesn't work well with file downloads, and we do return correct content type. The sandbox serves valid PDF when hit directly from postman.